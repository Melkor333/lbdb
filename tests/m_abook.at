dnl Copyright (c) 2018 Roland Rosenfeld <roland@spinnaker.de>
dnl
dnl This program is free software; you can redistribute it and/or modify
dnl it under the terms of the GNU General Public License as published by
dnl the Free Software Foundation; either version 2 of the License, or
dnl (at your option) any later version.
dnl
dnl This program is distributed in the hope that it will be useful,
dnl but WITHOUT ANY WARRANTY; without even the implied warranty of
dnl MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
dnl GNU General Public License for more details.
dnl
dnl You should have received a copy of the GNU General Public License
dnl along with this program; if not, write to the Free Software Foundation,
dnl Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.

AT_BANNER([Test m_abook])

AT_SETUP(m_abook)

AT_SKIP_IF([! which abook])
AT_CHECK([cat $abs_srcdir/data/m_abook.expected > expout])
AT_CHECK([env LBDB_OVERRIDE_METHODS=m_abook \
	      OVERRIDE_ABOOK_FILES=$abs_srcdir/data/abook.addressbook \
	  lbdbq First1], [], [expout])

AT_CLEANUP
