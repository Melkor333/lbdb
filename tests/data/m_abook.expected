lbdbq: 2 matches
foo1@example.com	First1 Family1	abook
foo1@example.net	First1 Family1	abook
